reference/polkadot-hub/assets.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Polkadot Hub Assets
+description: Learn about asset management on Polkadot Hub, including on-chain assets, foreign asset integration, and XCM for cross-chain asset transfers.
+categories: Polkadot Protocol
+---
+
+# Assets on Polkadot Hub
+
+## Introduction
+
+Polkadot Hub is Polkadot's system parachain for issuing and managing on-chain assets. While the relay chain provides security, Polkadot Hub handles asset logic—minting, burning, transfers, and metadata—efficiently and cost-effectively.
```

Comment:
This can be misread as if this is all that Polkadot Hub does. If we can phrase it slightly different. Assets are one of the several responsibilities of Polkadot Hub. Not the sole responsibility.